README.md

```diff
@@ -279,18 +279,18 @@ Open `x64 Native Tools Command Prompt for VS2019` and run the following commands
 ``` batch
 > git config --global core.eol lf
 > git config --global core.autocrlf input
-> set OPENMODELICAHOME=
-> set OPENMODELICALIBRARY=
 > md dynawo-project && cd dynawo-project
-> git clone https://github.com/dynawo/dynawo.git dynawo
+> git clone https://github.com/dynawo/dynawo.git
 > cd dynawo
-> cmake -S dynawo/3rdParty -B b-3-p -DCMAKE_INSTALL_PREFIX=../d-3-p -DOPENMODELICA_INSTALL=../OpenModelica/Install -DOPENMODELICA_SRC=../OpenModelica/Source -DOMDEV_HOME=../OMDev -G "NMake Makefiles"
-> cmake --build b-3-p
-> cmake -S dynawo -B b -DCMAKE_INSTALL_PREFIX=../d-i -DDYNAWO_HOME=.. -DINSTALL_OPENMODELICA=../../OpenModelica/Install -DDYNAWO_THIRD_PARTY_DIR=../../d-3-p -G "NMake Makefiles
-> cmake --build b --target install
-> cmake --build b --target models
-> cmake --build b --target solvers
+> util\windows\dynawo.cmd build
 ```
+**Tip** If you already installed OpenModelica, it is possible to set the following environment variables to use the existing OpenModelica installation and avoid recompiling OpenModelica again :
```

Comment:
It needs to be an OpenModelica compiled by Dynawo so the sentence needs precision otherwise if it would cause issues and I would remove it

Reply:
I would propose to remove it as it might confuse beginners with dynawo who might try to point toward a standard version of OM.